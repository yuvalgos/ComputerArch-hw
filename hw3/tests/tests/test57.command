./dflow_calc tests/test57.dat tests/test57.in d139 d147 p190 p166 p156 d138 d63 p124 d79 d155 p101 d186 d31 p37 p42 d133 d114 p32 p66 p199 p187 d102 d149 p18 p17 p48 d61 d3 d93 p41 p3 d146 p147 d161 p4 p131 d104 p8 d17 d24 d90 p5 d15 d29 p54 p32 d30 d116 p113 p163 p194 p173 p2 p81 p21 d72 d181 d80 d171 d113 p71 d16 p162 p170 p189 p98 p132 p82 d22 p110 p79 d36 d142 d118 p46 p75 d75 d34 p20 d130 p165 p71 d2 p29 p195 p33 p147 d117 d103 d156 d88 d177 d105 p91 p128 d74 d165 d74 d102 p125 p198 p139 p131 p33 d81 d54 d39 d46 d7 p151 p6 d93 d69 d52 p7 d39 d66 p27 d124 p52 d138 p127 p83 p105 d146 p7 p196 d146 p35 p47 p21 p64 d175 p27 p34 p31 d32 p55