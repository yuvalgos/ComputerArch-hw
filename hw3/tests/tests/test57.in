2 31 13 10
6 13 6 1
2 21 31 15
8 4 8 25
6 19 17 18
4 22 10 0
6 8 15 24
6 0 0 9
1 15 8 28
5 7 11 15
2 31 15 26
8 30 19 22
0 19 13 14
6 8 27 10
6 23 18 30
6 23 9 2
1 11 1 15
3 14 0 2
3 10 4 24
6 9 14 22
0 1 4 23
0 7 14 26
2 12 1 0
0 9 0 22
7 16 3 0
2 28 31 27
0 18 8 16
5 25 28 20
7 17 2 13
1 22 10 3
1 19 24 23
1 1 16 18
5 14 24 20
4 4 20 25
5 2 5 21
1 13 16 20
0 6 11 30
2 27 12 3
5 30 8 21
2 14 8 24
7 8 8 8
3 25 8 23
0 30 4 8
6 28 15 26
2 20 20 18
1 21 18 19
4 31 19 15
2 10 28 0
0 17 27 20
4 4 14 24
6 16 19 8
1 23 26 14
2 8 28 27
0 29 6 9
2 0 18 7
5 29 19 4
6 28 1 31
4 2 27 4
7 17 22 22
0 2 0 21
4 11 2 20
0 28 6 5
2 29 19 24
5 28 24 10
8 29 25 2
8 6 21 27
7 11 2 27
4 9 23 4
3 8 3 4
8 0 28 4
6 15 3 4
3 14 22 9
5 30 16 30
6 18 13 23
6 6 20 27
0 2 21 16
3 14 14 15
0 17 22 5
3 31 16 7
7 15 10 5
6 11 16 2
4 20 26 31
2 9 25 25
7 6 20 24
2 22 15 25
1 1 10 27
6 14 6 28
0 6 11 4
0 5 23 11
4 10 23 15
5 29 8 1
5 19 23 25
7 22 8 17
5 7 18 11
1 18 5 6
1 5 3 11
1 19 21 7
1 15 4 11
6 31 0 10
6 24 14 12
3 25 2 5
3 23 8 1
6 24 0 16
5 8 14 14
5 11 26 0
5 29 10 9
6 20 17 16
7 4 12 26
4 14 24 19
1 11 5 23
2 3 8 28
3 7 23 23
3 0 31 4
0 6 10 23
4 7 3 10
1 17 7 23
6 17 5 24
5 27 30 24
3 22 11 23
8 26 23 20
8 3 26 1
3 30 27 27
6 19 18 9
5 21 14 0
1 17 21 27
3 24 9 31
6 6 23 17
0 19 26 20
5 25 13 9
5 21 23 7
6 7 23 21
8 12 16 6
2 5 15 6
6 23 7 2
0 6 3 4
5 3 23 16
4 21 13 22
4 2 4 20
2 8 8 15
5 1 23 6
8 24 6 12
8 17 0 31
7 27 16 10
0 30 23 1
4 24 0 17
3 26 10 16
4 28 2 11
2 8 18 0
3 7 7 26
2 16 18 12
0 16 22 9
0 7 14 8
8 28 8 27
0 25 5 9
7 0 27 25
2 30 4 28
3 25 29 14
4 29 17 24
4 24 21 4
7 6 1 21
2 2 16 1
0 24 15 4
8 5 4 24
1 20 24 18
4 29 0 23
6 5 26 15
0 12 5 14
7 2 4 30
8 3 13 24
2 24 21 1
3 18 31 24
8 26 0 12
0 6 19 10
3 14 14 14
5 26 23 14
6 4 11 25
8 25 6 7
8 8 22 7
6 2 26 23
4 3 25 10
2 8 4 24
6 22 9 2
8 14 15 31
3 14 27 10
2 12 17 16
5 14 0 4
6 28 12 17
1 18 9 26
0 4 25 9
0 1 19 0
7 19 9 11
1 3 1 20
0 4 5 22
2 24 9 18
7 31 6 14
1 0 15 14
6 29 3 28
2 8 12 5
5 13 10 24
2 10 23 1
