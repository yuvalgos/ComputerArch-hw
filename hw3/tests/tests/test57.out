Reading the opcodes latency info from tests/test57.dat ... Got latency for 9 opcodes
Reading the program file tests/test57.in ... Found 200 instructions
getProgDepth()==232
getInstDeps(139)=={133,134}
getInstDeps(147)=={94,112}
getDepDepth(190)==78
getDepDepth(166)==106
getDepDepth(156)==106
getInstDeps(138)=={103,97}
getInstDeps(63)=={-1,47}
getDepDepth(124)==119
getInstDeps(79)=={47,-1}
getInstDeps(155)=={107,152}
getDepDepth(101)==45
getInstDeps(186)=={184,141}
getInstDeps(31)=={24,26}
getDepDepth(37)==30
getDepDepth(42)==54
getInstDeps(133)=={130,75}
getInstDeps(114)=={110,89}
getDepDepth(32)==0
getDepDepth(66)==45
getDepDepth(199)==183
getDepDepth(187)==196
getInstDeps(102)=={69,50}
getInstDeps(149)=={94,131}
getDepDepth(18)==23
getDepDepth(17)==17
getDepDepth(48)==43
getInstDeps(61)=={36,-1}
getInstDeps(3)=={-1,-1}
getInstDeps(93)=={73,80}
getDepDepth(41)==36
getDepDepth(3)==0
getInstDeps(146)=={137,109}
getDepDepth(147)==102
getInstDeps(161)=={97,107}
getDepDepth(4)==0
getDepDepth(131)==139
getInstDeps(104)=={-1,69}
getDepDepth(8)==17
getInstDeps(17)=={7,-1}
getInstDeps(24)=={-1,7}
getInstDeps(90)=={68,85}
getDepDepth(5)==0
getInstDeps(15)=={-1,-1}
getInstDeps(29)=={18,-1}
getDepDepth(54)==38
getDepDepth(32)==0
getInstDeps(30)=={-1,15}
getInstDeps(116)=={95,102}
getDepDepth(113)==72
getDepDepth(163)==77
getDepDepth(194)==178
getDepDepth(173)==106
getDepDepth(2)==5
getDepDepth(81)==60
getDepDepth(21)==24
getInstDeps(72)=={50,42}
getInstDeps(181)=={82,178}
getInstDeps(80)=={50,75}
getInstDeps(171)=={154,166}
getInstDeps(113)=={89,101}
getDepDepth(71)==44
getInstDeps(16)=={-1,8}
getDepDepth(162)==77
getDepDepth(170)==201
getDepDepth(189)==165
getDepDepth(98)==78
getDepDepth(132)==139
getDepDepth(82)==43
getInstDeps(22)=={20,7}
getDepDepth(110)==120
getDepDepth(79)==42
getInstDeps(36)=={16,11}
getInstDeps(142)=={50,89}
getInstDeps(118)=={109,101}
getDepDepth(46)==20
getDepDepth(75)==53
getInstDeps(75)=={45,50}
getInstDeps(34)=={-1,2}
getDepDepth(20)==23
getInstDeps(130)=={101,129}
getDepDepth(165)==79
getDepDepth(71)==44
getInstDeps(2)=={0,-1}
getDepDepth(29)==30
getDepDepth(195)==178
getDepDepth(33)==43
getDepDepth(147)==102
getInstDeps(117)=={72,102}
getInstDeps(103)=={86,86}
getInstDeps(156)=={105,108}
getInstDeps(88)=={51,80}
getInstDeps(177)=={118,151}
getInstDeps(105)=={89,82}
getDepDepth(91)==43
getDepDepth(128)==48
getInstDeps(74)=={44,37}
getInstDeps(165)=={145,97}
getInstDeps(74)=={44,37}
getInstDeps(102)=={69,50}
getDepDepth(125)==95
getDepDepth(198)==204
getDepDepth(139)==183
getDepDepth(131)==139
getDepDepth(33)==43
getInstDeps(81)=={-1,78}
getInstDeps(54)=={26,21}
getInstDeps(39)=={13,-1}
getInstDeps(46)=={30,8}
getInstDeps(7)=={-1,-1}
getDepDepth(151)==107
getDepDepth(6)==0
getInstDeps(93)=={73,80}
getInstDeps(69)=={63,49}
getInstDeps(52)=={43,37}
getDepDepth(7)==0
getInstDeps(39)=={13,-1}
getInstDeps(66)=={59,37}
getDepDepth(27)==30
getInstDeps(124)=={123,117}
getDepDepth(52)==37
getInstDeps(138)=={103,97}
getDepDepth(127)==93
getDepDepth(83)==71
getDepDepth(105)==72
getInstDeps(146)=={137,109}
getDepDepth(7)==0
getDepDepth(196)==170
getInstDeps(146)=={137,109}
getDepDepth(35)==36
getDepDepth(47)==37
getDepDepth(21)==24
getDepDepth(64)==45
getInstDeps(175)=={109,156}
getDepDepth(27)==30
getDepDepth(34)==10
getDepDepth(31)==38
getInstDeps(32)=={-1,-1}
getDepDepth(55)==33
